### PR TITLE
Add Card primitive

### PR DESCRIPTION
**Pull request by bastien-hugon**

### CODE REVIEW

File: packages/streamline/src/theme/shadows.ts [hunk 1]
```diff
@@ -27,6 +27,10 @@ export const shadows = {
     },
     shadowRadius: 20,
     shadowOpacity: 0.12,
-    elevation: 4,
+    elevation: 12,
```
Comment (kimak) [inline packages/streamline/src/theme/shadows.ts:30]: keep 4?

File: packages/streamline/src/theme/index.ts [hunk 2]
```diff
@@ -32,6 +39,7 @@ export const theme = createTheme({
     opacityDuration,
     onPressScale,
   },
+  shadows: shadows,
```
Comment (kimak) [inline packages/streamline/src/theme/index.ts]: I think you can remove from here

File: packages/sandbox/e2e/visual_regressions_flow.yml [hunk 3]
```diff
@@ -32,6 +32,12 @@ appId: ${APPID}
     timeout: 5000
 - takeScreenshot: e2e/${PLATFORM}/tempScreenshots/ButtonMini
 - tapOn: 'Button Mini'
+- tapOn: 'Card >'
+- extendedWaitUntil:
+    visible: 'Card'
+    timeout: 5000
```
Comment (kimak) [inline packages/sandbox/e2e/visual_regressions_flow.yml]: Use disableAnimation context instead?

**Review by kimak — APPROVED** [review]

File: packages/streamline/src/theme/shadows.ts (hunk 1 above)
Comment (bastien-hugon) [reply packages/streamline/src/theme/shadows.ts:30]: It looks like the figma with 12... We need to align with IAN

File: packages/streamline/src/theme/animation.ts [hunk 4]
```diff
@@ -1,32 +1,48 @@
...
+  end: disableAnimation ? 0.95 : 0.95,
+  duration: disableAnimation ? 250 : 250,
+});
+
+export const getAnimationValues = (disableAnimation: boolean) => ({
+  appearDuration: getAppearDuration(disableAnimation),
+  appearEntering: getAppearEntering(disableAnimation),
+  appearExiting: getAppearExiting(disableAnimation),
+  opacityDuration: getOpacityDuration(disableAnimation),
+  onPressScale: getOnPressScale(disableAnimation),
```
Comment (kimak) [inline packages/streamline/src/theme/animation.ts]: `onPressScale: disableAnimation ? getOnPressScale() : undefined`

To avoid leak into API functions

**Pull request merged**